### DISCUSSION TITLE
laravel new <projectname> command fails with php version PHP 8.3.0alpha2 (works fine with php v 8.2)

### DISCUSSION THREAD

**Discussion by connecteev** [laravel/framework#47891] [Q&A]
### Laravel Version

10.2.5

### PHP Version

8.3.0alpha2

### Description

laravel new kb_boilerplate_new

   _                               _
  | |                             | |
  | |     __ _ _ __ __ ___   _____| |
  | |    / _` | '__/ _` \ \ / / _ \ |
  | |___| (_| | | | (_| |\ V /  __/ |
  |______\__,_|_|  \__,_| \_/ \___|_|

Creating a "laravel/laravel" project at "./kb_boilerplate_new"
Installing laravel/laravel (v10.2.5)
  - Installing laravel/laravel (v10.2.5): Extracting archive
Created project in /Users/kunalpunjabi/Code/web/__Herd/kb_boilerplate_new
> @php -r "file_exists('.env') || copy('.env.example', '.env');"
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - laravel/framework[v10.10.0, ..., v10.16.1] require league/commonmark ^2.2.1 -> satisfiable by league/commonmark[2.2.1, ..., 2.4.0].
    - league/commonmark[2.2.1, ..., 2.4.0] require league/config ^1.1.1 -> satisfiable by league/config[v1.1.1, v1.2.0].
    - league/config[v1.1.1, ..., v1.2.0] require nette/schema ^1.2 -> satisfiable by nette/schema[v1.2.0, v1.2.1, v1.2.2, v1.2.3].
    - nette/schema[v1.2.0, ..., v1.2.1] require php >=7.1 <8.1 -> your php version (8.3.0alpha2) does not satisfy that requirement.
    - nette/schema v1.2.2 requires php >=7.1 <8.2 -> your php version (8.3.0alpha2) does not satisfy that requirement.
    - nette/schema v1.2.3 requires php >=7.1 <8.3 -> your php version (8.3.0alpha2) does not satisfy that requirement.
    - Root composer.json requires laravel/framework ^10.10 -> satisfiable by laravel/framework[v10.10.0, ..., v10.16.1].

### Steps To Reproduce

- Use PHP 8.3.0alpha2
- Run: laravel new < projectname >
- Command fails.
- Note: This command works fine with php 8.2

**Comment by crynobone** [comment]
Hi there,

`nette/schema`  currently prevent installation on PHP 8.3 and we need to wait for now.

**Reply by connecteev** [reply]
Thanks, this isn't blocking me, just thought I would report
